bot/cogs/todo_list.py

```diff
@@ -1,20 +1,23 @@
 import json
 import os
+import logging
 from asyncio import sleep
 from pathlib import Path
 from bot.settings import BASE_DIR
 from discord.ext import tasks
 from discord.ext.commands import Bot, Cog, command
 from discord import Embed, Colour
 
+logger = logging.getLogger(__name__)
+
 
 async def update_of_todos(ctx, todos):
     """This will create a todo list from todos"""
 
     todos = [t.strip() for t in todos.split(",") if t.strip() != ""]
 
     if Path(f"{BASE_DIR}/todo_list_data.json").is_file():  # Checks file if it exists
-        print("File exists")
+        logger.info("File exists")
```

Comment:
What file exists? I think these need to be more specific, along with the other vague file existing/not existing info logs in this file.